src/App.js

```diff
@@ -31,7 +31,7 @@ function App() {
             <div className={styles.parent}>
                 <GlobalStyles/>
                 <Header changeTheme={changeTheme}/>
-                <Main/>
+                <VideoPage/>
```

Comment:
videopage is not reusable

Reply:
bro u really suck, we can not use props

Reply:
fixed, sometimes u suck, sometimes u monke